whisper/cli/src/main.rs

```diff
@@ -226,7 +243,10 @@ fn execute<S, I>(command: I) -> Result<(), Error> where I: IntoIterator<Item=S>,
 	let pool_size = args.flag_whisper_pool_size * POOL_UNIT;
 	let rpc_url = format!("{}:{}", args.flag_rpc_address, args.flag_rpc_port);
 
-	initialize_logger(args.flag_log);
+	// Default values are defined in the CLI configuration
+	let logging_level = args.clone().flag_logging;
+	let logging_to_file = args.clone().flag_logging_to_file;
+	config::logger::init_logger(&logging_level, logging_to_file);
```

Comment:
Why not `config::logger::init_logger(args.flag_logging, args.flag.logging_to_file)`?

Reply:
Yes that works! I've pushed changes in commit c1646d94c39644501286662dcf173f05ed38b503